tests/integration/conftest.py

```diff
@@ -43,8 +43,5 @@ def os_indendent_tmp():
         an_tmp = os.path.join(tmp_real, "an")
     else:
         an_tmp = os.path.join("/tmp", "private", "an")
-    os.makedirs(an_tmp)
-    try:
-        yield an_tmp
-    finally:
-        shutil.rmtree(an_tmp)
```

Comment:
It might still be good to clean up after ourselves, otherwise when iterating locally, we can potentially end up accumulating stuff. For now, it's probably not a huge issue, but in the future if we start using the directory for more, it could be.

I'm guessing this is solving a race condition? It might be better to use per-test tmpdirs instead in that case, and still keep the removal logic. That way tests don't step on each other, but we can still nuke the directories after.

Reply:
we needed a directory of predictable length between dev boxes and CI so the column widths don't vary, I'll add a rm to tox, yes because with xdist it was getting deleted during other test runs